Resources/Prototypes/_NF/Entities/Clothing/Back/base_clothing_backpack.yml

```diff
@@ -28,6 +28,10 @@
     size: Huge
     shape:
     - 0,0,7,5
+  - type: ClothingSpeedModifier
+    walkModifier: 1
+    sprintModifier: 0.9
+  - type: HeldSpeedModifier
```

Comment:
Probably best to reparent from ClothingBackpackDuffel vs. ClothingBackpack, you'll get these "for free" (unless the explicit goal was to completely detach from upstream duffel), and everything else is redefined.